app/helpers/application_helper.rb

```diff
@@ -165,17 +165,14 @@ def active_nav_li(link)
   end
 
   def show_time(length)
-    h = length / 60
-    min = length - h * 60
+    h, min = length.divmod(60)
```

Comment:
That's a very good question. I was initially thinking we could just throw a nil there, but in the unlikely event that things miserably break, if we show 0h 0m it would make it visually easier to find the failure.
Right now we use this function for the event_type.length in the dropdown of proposal#new

So 0h 0m ?